### ISSUE TITLE
Comment based card broken in MarkDown

### ISSUE THREAD

**Issue by telenieko** [imdone/imdone#41]
Parsing of comment based cards in markdown files is broken in 1.6.24

Using this example project:

```markdown
# [MD based Card](#NOTE:0)
<!--
created:2021-01-19T09:34:20.681Z
-->

<!-- #TODO:20 Comment based card in MD
+urgent
-->

<!-- #DONE:0 Done Comment based card in MD
+urgent completed:2021-01-19T09:35:23.603Z
-->
```

In version 1.6.10 renders like:
![Screenshot 2021-01-19 at 10 35 54](https://user-images.githubusercontent.com/10505/105016118-8d4e6a80-5a42-11eb-8c26-964df7f78473.png)

The same file in 1.6.24 renders like:
![Screenshot 2021-01-19 at 10 37 06](https://user-images.githubusercontent.com/10505/105016150-963f3c00-5a42-11eb-84bd-0c4cff24c925.png)

Notice that the cards based on comments do no longer show any text.

I have gone back up to 1.6.22 without improvement.

More than half of my cards are comment based :'(

**Comment by piascikj** [comment]
I've got a fix prepared and will have it implemented in 1.6.25 later today.

**Comment by piascikj** [comment]
This is fixed in 1.6.25
